bridge/client/app/_components/ktb-project-settings-git-extended/ktb-project-settings-git-extended.component.ts

```diff
@@ -105,6 +111,10 @@ export class KtbProjectSettingsGitExtendedComponent implements OnInit {
       this.dataService.updateGitUpstream(this.projectName, this.gitData).subscribe(
```

Comment:
Shouldn't this subscription be unsubscribed at some point? Might be a dangling subscription, did not find anything in the documentation on HttpClient, though...

Reply:
``OK, i saw it in the source: https://github.com/angular/angular/blob/986e80e9db46a267d33667464bb64c3501f8c72b/packages/common/http/src/xhr.ts#L193

```
        if (ok) {
          // A successful response is delivered on the event stream.
          observer.next(new HttpResponse({
            body,
            headers,
            status,
            statusText,
            url: url || undefined,
          }));
          // The full body has been received and delivered, no further events
          // are possible. This request is complete.
          observer.complete();
        } else {
```